tests/Foliage/Tests/Utils.hs

```diff
@@ -0,0 +1,47 @@
+module Foliage.Tests.Utils (
+  checkRequiredProgram,
+  callCommand,
+  readCommand,
+  withFixture,
+) where
+
+import Control.Exception (bracket)
+import Control.Monad (when)
+import Data.Foldable (for_)
+import Data.Functor (void)
+import Data.Maybe (isNothing)
+import System.Directory
+import System.FilePath
+import System.Posix.Temp (mkdtemp)
+import System.Process (readCreateProcess, shell)
+
+-- | Set up a temporary directory prepopulated with the fixture files
+withFixture :: FilePath -> IO () -> IO ()
+withFixture name =
+  bracket acquire release . const
+ where
+  acquire = do
+    cur <- getCurrentDirectory
+    fixtureDir <- makeAbsolute name
+    workDir <- mkdtemp $ fixtureDir ++ "."
```

Comment:
why the dot?

Reply:
I liked it better :joy: `tests/fixtures/simple123423` vs `tests/fixtures/simple.123423`

Reply:
oh I see. worth a comment for bewildered readers such as myself